packages/nodes-base/nodes/MongoDb/MongoDb.node.ts

```diff
@@ -208,6 +210,10 @@ export class MongoDb implements INodeType {
 						query = query.sort(sort);
 					}
 
+					if (projection && projection instanceof Document) {
```

Comment:
Hi @Joffcom, I'm sorry but it looks like this object is never imported and came from bson module (which is not a direct dependency).

I'm wondering why calling a projection throws a "Document is not defined" error, but not all the previous calls in this file : what do I have done wrong, here ? :/